pkg/kinds/accesspolicy/accesspolicy_gen.go

```diff
@@ -30,11 +30,10 @@ func NewK8sResource(name string, s *Spec) K8sResource {
 	}
 }
 
-
 // Resource is the wire representation of AccessPolicy.
 // It currently will soon be merged into the k8s flavor (TODO be better)
 type Resource struct {
 	Metadata Metadata `json:"metadata"`
-	Spec Spec `json:"spec"`
-	Status Status `json:"status"`
+	Spec     Spec     `json:"spec"`
+	Status   Status   `json:"status"`
```

Comment:
looks like a formatter was run over these after codegen, which is gonna cause CI to fail, because generated code must be byte-equal to the original output of code generation.

fix is to rerun `make gen-cue` and commit the result without any further modifications.